numpy/core/_utils.py

```diff
@@ -0,0 +1,14 @@
+import warnings
+
+
+def _raise_warning(attr: str) -> None:
+
+    warnings.warn(
+        f"`numpy.core` has been made officially private and renamed to "
+        "`numpy._core`. Accessing `_core` directly is discouraged, you "
+        "should use a public module instead that exports the attribute "
+        "in question. If you still would like to access an attribute "
+        f"from it, please use `np._core.{attr}`.",
```

Comment:
Also add a line warning them that things in `np._core` can change without warning at any time.

Reply:
Good point - I added a line that says it can change without a warning.